scripts/selinux/README.md

```diff
@@ -36,16 +44,42 @@ sudo restorecon -R /etc/nginx-agent
 ```
 
 ## Updating existing policy
-Copy the folder `scripts/selinux` over to your rhel server.
+Copy the folder `scripts/selinux` over to your rhel 8 server.
 Then run the following command to update the policy:
 ```
 sudo ./nginx_agent.sh --update
 ```
+Then copy the `nginx_agent.te` and `nginx_agent.pp` files back and create a PR with the changes.
```

Comment:
Is it worth adding to check the policy doesn't need further updates with
`sudo ausearch -m avc --raw -se nginx_agent | audit2allow -R`  before you copy back the files and make a PR ?

Reply:
The nginx_agent.sh script already handles that. See here https://github.com/nginx/agent/blob/main/scripts/selinux/nginx_agent.sh#L14 & here https://github.com/nginx/agent/blob/main/scripts/selinux/nginx_agent.sh#L17

Reply:
ok was just thinking that we ran it to check last time so was wondering if it needed to be in the doc